Modules/input.py

```diff
@@ -270,14 +270,15 @@ def Decode8000_v2(self, Devices, MsgData, MsgRSSI) : # Status
         loggingInput( self, 'Log',"Decode8000 - uncomplete message : %s" %MsgData)
         return
 
-    if MsgLen > 8 :
-        loggingInput( self, 'Log',"Decode8000 - More information . New Firmware ???")
-        loggingInput( self, 'Log',"Decode8000 - %s" %MsgData)
-
     Status=MsgData[0:2]
     SEQ=MsgData[2:4]
     PacketType=MsgData[4:8]
 
+    if MsgLen > 8 :
```

Comment:
(1) The SQN management should be done in Transport, no ?
(1) Shouldn't SQNack set to None if len is < 10, and SQNSack set to the real APSack when len is equal to 10 ?